mongodb/vibe/db/mongo/settings.d

```diff
@@ -321,6 +321,7 @@ class MongoClientSettings
 
 	string username;
 	string digest;
+	uint max_connections = uint.max;
```

Comment:
It is because `uint.max` is default value for ConnectionPool ctor argument `max_concurrent`